lib/temporal/testing/local_workflow_context.rb

```diff
@@ -170,12 +170,22 @@ def wait_for_all(*futures)
         return
       end
 
-      def wait_for(*futures, &unblock_condition)
-        if futures.empty? && unblock_condition.nil?
-          raise 'You must pass either a future or an unblock condition block to wait_for'
+      def wait_for_any(*futures)
+        return if futures.empty?
+
+        while futures.empty? || futures.none?(&:finished?)
+          Fiber.yield
+        end
+
+        return
+      end
+
+      def wait_until(&unblock_condition)
```

Comment:
I suppose I could also keep calling it `wait_for` and have it only take the condition block. That would be breaking, and I could add an explanatory comment

Reply:
I think approach here is clearer. It seems better to have the method completely go away, so that any breakage would be resolved by looking for this PR or by reading comments on workflow context.